### ISSUE TITLE
shape_impl.rs lacks a Capsule impl

### ISSUE THREAD

**Issue by BillyIII** [rustsim/ncollide#175]
Shape seems to be implemented for a lot of primitives, but not for a Capsule. Is there a reason, or is it just an oversight?

**Comment by sebcrozet** [comment]
Only an oversight. Mostly because I did not take the time to implement all the relevant traits yet.

**Comment by sebcrozet** [comment]
The implementation of collision detection involving a capsule will be very straightforward once the changes made to the narrow-phase system land with the new support for deformable collision objects. Therefore I will postpone this until the work on deformable collision objects lands (in two or three months I guess).
